paypro-gateways-woocommerce/includes/paypro/api/PayProApiHelper.php

```diff
@@ -41,10 +47,15 @@ public function getSaleStatus($payment_hash)
 
     private function execute()
     {
-        if($this->testMode) $this->api->setParam('test_mode', 'true'); else $this->api->setParam('test_mode', 'false');
+        $this->api->setParam('test_mode', ($this->testMode ? 'true' : 'false'));
 
         $result = $this->api->execute();
 
+        if($result['return'] == self::PAYPRO_API_RES_APIKEY_INVALID) {
+            $result['errors'] = 'true';
```

Comment:
Do we also want to set a message here?

Reply:
There is already a message given by the api itself `API key not valid` which will then also go thru the new error handling which will end up as `There was an issue processing your payment.` in the frontend